### PR TITLE
Prefetch current user

### PR DESCRIPTION
**Pull request by yuliaveres**

### CODE REVIEW

File: Token/Controllers/Settings/ProfileController.swift [hunk 1]
```diff
@@ -278,7 +278,9 @@ open class ProfileController: UIViewController {
     }
 
     fileprivate func updateReputation() {
-        RatingsClient.shared.scores(for: TokenUser.current!.address) { ratingScore in
```
Comment (Elland) [inline Token/Controllers/Settings/ProfileController.swift]: 👏🏽

File: Token/Models/TokenUser.swift [hunk 2]
```diff
@@ -146,6 +137,21 @@ public class TokenUser: NSObject, NSCoding {
         ]
     }
 
+    public static func retrieveCurrentUser() {
+        if let userData = (Yap.sharedInstance.retrieveObject(for: TokenUser.storedUserKey) as? Data), self._current == nil,
```
Comment (Elland) [inline Token/Models/TokenUser.swift]: I think it's clearer if you remove this from the `if let` call and do a `guard` beforehand. 

```swift
guard self._current == nil else { return }
if let ... {}
```

File: Token/Supporting Files/AppDelegate.m [hunk 3]
```diff
@@ -62,6 +62,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(createOrRestoreNewUser) name:@"CreateNewUser" object:nil];
 
     [self configureAndPresentWindow];
+    [TokenUser retrieveCurrentUser];
```
Comment (Elland) [inline Token/Supporting Files/AppDelegate.m]: 👌🏽

**Comment by Elland** [comment]
Looks solid on my end. Merging.

File: Token/Models/TokenUser.swift (hunk 2 above)
Comment (yuliaveres) [inline Token/Models/TokenUser.swift]: oh, sorry, this is odd at all, we check inside current getter the same, so not needed here

**Pull request merged**